1주차/10773.py

```diff
@@ -0,0 +1,11 @@
+k=int(input())
+stk=[]
+
+for i in range(k):
+    a=int(input())
+    if(a==0):
```

Comment:
```suggestion
    if not a:
```
`a == 0` 도 좋지만 `not a` 로 쓸 수도 있다는것도 알아만 두세요
그리고 파이썬 if문엔 괄호 안써도 됩니다~